frontend/ui/uimanager.lua

```diff
@@ -218,6 +219,8 @@ function UIManager:close(widget, refreshtype, refreshregion)
     local dirty = false
     -- first send close event to widget
     widget:handleEvent(Event:new("CloseWidget"))
+    -- Ensure all the widgets can get onFlushSettings event.
+    widget:handleEvent(Event:new("FlushSettings"))
```

Comment:
Reasonable. Though currently we do not have conflicts between CloseWidget and FlushSettings.